doc/vim-jsonnet.txt

```diff
@@ -0,0 +1,105 @@
+*vim-jsonnet.txt* Jsonnet development plugin
+*vim-jsonnet*
+
+
```

Comment:
Lots of whitespace at the top, and I guess you want some under the banner?  I presume this is not a requirement of the format of this file (I've never written vim docs before).